### ISSUE TITLE
The package does not appear to be on npm

### ISSUE THREAD

**Issue by kontur** [Monotype/variableFont.js#6]
As simple as the title, the package does not actually appear to be in the npm registry.

**Comment by codelastnight** [comment]
Hey, I had the same issue. I found the [package listing online](https://www.npmjs.com/package/@kennethormandy/variablefont.js), and turns out the installation has changed to 
```npm i @kennethormandy/variablefont.js```
instead. I know I'm a year late but I hope this helps someone

**Comment by kontur** [comment]
Yep, that appears to be the fork by @kennethormandy — the issue is still valid for this repo, if indeed anybody is ever maintaining this code (:
